cfgov/scripts/http_smoke_test.py

```diff
@@ -0,0 +1,233 @@
+]
+
+SHORT_RUN = [
+    '/',
+    '/es/',
+    '/complaint/',
+    '/learnmore/',
+    # '/askcfpb/',
+    '/askcfpb/search',
+    '/your-story/',
+    '/students/',
+    '/find-a-housing-counselor/',
+    '/older-americans/',
+    # '/servicemembers/',
+    '/consumer-tools/auto-loans/',
+    '/paying-for-college/',
+    '/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/',
+    '/owning-a-home/',
+    '/retirement/before-you-claim/',
+    # '/retirement/before-you-claim/es/',
+    '/sending-money/',
+    '/know-before-you-owe/',
+    # '/mortgagehelp/',
+    '/fair-lending/',
+    # '/your-money-your-goals/',
+    '/adult-financial-education/',
+    # '/youth-financial-education/',
+    '/library-resources/',
+    # '/tax-preparer-resources/',
+    '/money-as-you-grow/',
+    # '/empowerment/',
+    # '/managing-someone-elses-money/',
+    # '/data-research/',
+    # '/data-research/research-reports/',
+    # '/data-research/cfpb-research-conference/',
+    '/data-research/consumer-complaints/',
+    '/data-research/hmda/',
+    # '/data-research/credit-card-data/',
+    '/policy-compliance/',
+    # '/policy-compliance/rulemaking/',
+    # '/policy-compliance/guidance/',
+    # '/policy-compliance/enforcement/',
+    # '/policy-compliance/notice-opportunities-comment/',
+    # '/policy-compliance/amicus/',
+    # '/policy-compliance/guidance/implementation-guidance/',
+    '/about-us/',
+    '/about-us/careers/current-openings/',
+    # '/about-us/the-bureau/',
+    # '/about-us/budget-strategy/',
+    # '/about-us/payments-harmed-consumers/',
+    # '/about-us/blog/',
+    '/about-us/newsroom/',
+    # '/about-us/events/',
+    # '/about-us/careers/',
+    # '/about-us/doing-business-with-us/',
+    # '/about-us/advisory-groups/',
+    # '/about-us/project-catalyst/',
+    # '/about-us/contact-us/',
+    # '/about-us/the-bureau/',
+    '/activity-log/',
+]
+HELP = ("SMOKE TEST USAGE:\n    "
+            "Run this script with 'python {} [DOMAIN]'\n    "
+            "Domain options are 'www' 'pro1' and 'pro2'\n    "
+            "Using pro1 or pro2 will sidestep Akamai, but you "
+            "must run those tests from the EXT VPC\n    "
+            "Optional final param '--full' will increase the number "
+                "of URLs tested from {} to {}\n    "
+            "Using --full tests every link in our main nav\n    "
+            "Tests pass if the url returns status code 200")
+
+def check_urls(base, full=False):
+    """
+    A smoke test to make sure the main cfgov URLs are returning status 200.
+
+    Base options allow us to make http calls without hitting Akamai.
+    The 'full' option tests every link in the main cfgov nav.
+    """
+
+    base_url = BASES[base]
+    count = 0
+    timeouts = []
+    failures = []
+    starter = time.time()
+    if full:
+        url_list = FULL_RUN
+    else:
+        url_list = SHORT_RUN
+    for url_suffix in url_list:
+        # print(url_suffix)
```

Comment:
Can this comment be removed, or updated?  I could see it being helpful to log each URL, so maybe something like `logger.info(url_suffix)` instead?